testData/solutions/win32.json5

```diff
@@ -5797,6 +6462,498 @@
+                    "verifySettings": true
+                },
+                "supportedSettings": {
+                    "SwapMouseButtonsConfig": {
+                        "schema": {
+                            "title": "Swap mouse primary button",
+                            "description": "Swap mouse left/right buttons",
+                            "type": "boolean",
+                            "default": false
+                        }
+                    }
+                },
+                "capabilitiesTransformations": {
+                    "SwapMouseButtonsConfig": {
+                        "transform": {
+                            "type": "fluid.transforms.fallbackValue",
+                            "inputPath": "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.SwapMouseButtons",
+                            "outputPath": "value",
+                            "defaultValue": 0
+                        },
+                        "path": {
+                            "transform": {
+                                "type": "fluid.transforms.literalValue",
+                                "input": {
+                                    "get": "pvParam",
+                                    "set": "uiParam"
+                                }
+                            }
+                        }
+                    }
+                },
+                "inverseCapabilitiesTransformations": {
+                    "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.SwapMouseButtons": "SwapMouseButtonsConfig.value"
+                }
+            },
+            "configureScrollWheelMode": {
+                "type": "gpii.windows.spiSettingsHandler",
+                "liveness": "live",
+                "options": {
+                    "getAction": "SPI_GETWHEELSCROLLLINES",
+                    "setAction": "SPI_SETWHEELSCROLLLINES",
+                    "uiParam": 0,
+                    "pvParam": {
+                        "type": "UINT"
+                    },
+                    "fWinIni": "SPIF_UPDATEINIFILE|SPIF_SENDCHANGE",
+                    "verifySettings": true
+                },
+                "supportedSettings": {
+                    "ScrollWheelModeConfig": {
+                        "schema": {
+                            "title": "Mouse scroll wheel mode",
+                            "description": "Changes the way scroll wheel behaves, between lines and screens",
+                            "type": "number",
+                            "default": 3,
+                            "min": 0,
+                            "max": 25,
+                            "divisibleBy": 1,
+                            // This special value trigger the special meaning
+                            // of full-page scrolling.
+                            "specialValue": "'full-page' | 4294967295"
```

Comment:
This isn't a valid JSON schema element - what do you suggest, @the-t-in-rtf  ?

Reply:
I'm a bit confused about the intent here, but my guess is it's a list of integers, each of which corresponds to a particular setting?  If so, seems like `enum` and `enumLabels` are in order.

In any case `divisibleBy` needs to go, that's now `multipleOf`.

Reply:
Or perhaps that `specialValue` is the only supported literal value, in which case, still an `enum`.  If you can use that literal value but also integers, you'll need `oneOf` or something like that.

Reply:
Sure, @JavierJF, I'm around.  A little detail on what you're actually storing and how this special case relates would be a good starting point for the discussion.